tests/playwright/ui-tests/trusts/details-page.spec.ts

```diff
@@ -2,14 +2,17 @@ import { test } from '@playwright/test'
 import { DetailsPage } from '../../page-object-model/trust/details-page'
 import { NotFoundPage } from '../../page-object-model/not-found-page'
 import { FakeTestData } from '../../fake-data/fake-test-data'
+import { DataSourcePanelItem } from '../../page-object-model/trust/sources-and-updates-component'
 
 test.describe('Details page', () => {
   let detailsPage: DetailsPage
   let notFoundPage: NotFoundPage
+  const sources: DataSourcePanelItem[] = [{ fields: 'Trust details, Reference numbers:', dataSource: 'Get information about schools', update: 'Daily' }]
 
   test.beforeEach(async ({ page }) => {
     detailsPage = new DetailsPage(page, new FakeTestData())
     await detailsPage.goTo()
+    await detailsPage.sourcePanel.openPanel()
```

Comment:
This line shouldn't be needed as the `expect.toSeeCorrectSourceAndUpdates(sources)` opens the panel